docs/docs/en/guides/develop/cache.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: Cache
-titleTemplate: :title · Build · Develop · Guides · Tuist
+titleTemplate: :title · Develop · Guides · Tuist
 description: Optimize your build times by caching compiled binaries and sharing them across different environments.
 ---
 
 # Cache {#cache}
 
-> [!IMPORTANT] REMOTE PROJECT REQUIRED
-> This feature requires a <LocalizedLink href="/server/introduction/accounts-and-projects">remote project</LocalizedLink>.
+> [!IMPORTANT] REQUIREMENTS
+> - A <LocalizedLink href="/guides/develop/projects">Tuist Project</LocalizedLink>
+> - A <LocalizedLink href="/server/introduction/accounts-and-projects">serveraccount and project</LocalizedLink>
```

Comment:
```suggestion
> - A <LocalizedLink href="/server/introduction/accounts-and-projects">server account and project</LocalizedLink>
```

However, I'd change this to `Requirements: A Tuist account and project` and shift the mentions of "Tuist Project" to "Generated Project".